dev/devicelab/bin/tasks/module_test.dart

```diff
@@ -216,12 +216,12 @@ Future<void> main() async {
       }
 
       final String analyticsOutput = analyticsOutputFile.readAsStringSync();
-      if (!analyticsOutput.contains('cd24: android-arm64')
+      if (!analyticsOutput.contains('cd24: TargetPlatform.android')
```

Comment:
Is the real target architecture communicated in some other way to analytics?

Reply:
If not, we should figure out a way to retain that so we can understand our analytics across this change.

Reply:
It should be  part of the build apk/appbundle/aar commands:

https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/commands/build_apk.dart#L58

@blasten can you provide some additional context on why we send this second analytics call? Its for the add2app tracking, right?

Reply:
I don't think this is critical, but cc'ing @xster to confirm.

Reply:
ultranit: not having the enum 'outer class' name in there would be nice as a bonus. 

https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/commands/build_apk.dart#L58 that you're linking to is orthogonal to this right? This is to detect top level calls to `flutter build bundles` from add-to-app scripts

Reply:
the dashboard to monitor is https://dashboards.corp.google.com/_8211ecd6_e372_4f1c_b4f8_a1f4f0411822#xhaf6f3g0